deploy/ydb-operator/values.yaml

```diff
@@ -57,7 +57,15 @@ webhook:
     #  nodePort: 9443
     #
     ## Arbitrary fqdn for WebhookConfiguration instead of a default Service cluster fqdn:
-    #  fqdn: example.org
+    fqdn: example.org
+
+    ## PathPrefix for WebhookConfiguration url when fqdn used
+    ## Set variable to true to use default template <namespace>/<release> or
+    ## write yout own pathPrefix
+    pathPrefix: ""
```

Comment:
I have a suggestion. Maybe we don't merge the default and custom behaviour into one option, `pathPrefix`? It seems very counter intuitive. How about instead of one `pathPrefix` option we introduce two:
```
enableDefaultPathPrefix: false
customPathPrefix: "/myPrefix"
```

`enableDefaultPathPrefix` would be true by default, and having `customPathPrefix` set to something non-empty would override `enableDefaultPathPrefix`. 

Additionally, it would simplify the complex logic in the template with `kindIs` that you use to check that `pathPrefix` is bool. 

What do you think?

Reply:
Yeah, I think exactly the same, but have worried about two variables instead just one
I agree with you, it would be simpler and more intuitive